### PR TITLE
Add Vuex reference

### PR DESCRIPTION
**Pull request by Nicolapps**
This pull requests add a documentation set for [Vuex](https://vuex.vuejs.org), Vue.js’s state management library. It’s available under the MIT license as well.

Here’s the index of the documentation:
vuex

Here’s the Trello card: https://trello.com/c/71nataDX/378-vuex

### CODE REVIEW

File: public/icons/docs/vuex/SOURCE [hunk 1]
```diff
@@ -0,0 +1 @@
+http://vuejs.org/
```
Comment (gengjiawen) [inline public/icons/docs/vuex/SOURCE]: should this be https://vuex.vuejs.org/

Comment (Nicolapps) [reply public/icons/docs/vuex/SOURCE]: I’ve used the exact same icon than in the Vue.js documentation set, so it should probably be the same.

Comment (gengjiawen) [reply public/icons/docs/vuex/SOURCE]: I see, thanks.

**Comment by gengjiawen** [comment]
@Thibaut Can you take a look at this pr, thanks.

**Comment by Thibaut** [comment]
Sorry there are multiple PRs in the queue and I won't be able to merge this one in the very short-term. At the moment I'm working toward getting other contributors able to merge and deploy PRs, and everything transferred over to freeCodeCamp, while also keeping things updated. Once this is done, we'll work through the backlog of PRs.

**Comment by gengjiawen** [comment]
@Thibaut Thanks for the reply.

**Comment by gengjiawen** [comment]
@jmerle Can you take a look at this ? Thanks.

**Comment by jmerle** [comment]
@gengjiawen I'll be reviewing, modifying and merging all PR's adding scrapers that were not made by me (can't review myself for obvious reasons) this weekend.

**Comment by jmerle** [comment]
Thanks! Deployed to [https://devdocs.io/vuex/](https://devdocs.io/vuex/).

**Pull request merged**